msteams-platform/concepts/build-and-test/Shared-channels.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Shared channels
+author: Rajeshwari-v
+description:  Collaborating with shared channels.
+ms.author: surbhigupta
+localization_priority: Normal
+ms.topic: conceptual
+---
+
+# Shared channels
+
+Shared channels in Teams allow members of a channel to collaborate with users across other Teams and organizations. You can create and share a shared channel with:
+
+* Members of another Team within the same organization.
+* Individuals within the same organization.
+* Individuals and other Teams of other organizations.
+
+Shared channels facilitate collaboration seamlessly. Allows external users outside of your organization to collaborate with internal users in Teams without changing their user context. Enhances user experience unlike using guest accounts, for example, the members must sign out of Teams and sign in again using a guest account. Teams applications can now extend the powerful collaboration space.
+
+:::image type="content" source="~/assets/images/app-fundamentals/shared-channels-teams.png" alt-text="Shared Channel image"border="true" :::
+
+## Manifest update in shared channels
+
+When the content UX is loaded in a shared channel, use the data received from `getContext` call for  shared channel changes. `getContext` call publishes two new properties, `hostTeamGroupID` and `hostTenantID`, which are used to retrieve channel membership from the Microsoft Graph API. `hostTeam` is the team that created the shared channel.
+
+SupportedChannelTypes is an optional property that enables your app in non-standard channels. If your app supports the team scope and the property is defined, Teams will enable your app in each channel type accordingly. Private and shared channels are currently supported. For more information, see [supportedChannelTypes](../../resources/schema/manifest-schema.md#supportedchanneltypes)
```

Comment:
What are non-standard channels? Either link to an existing description (preferred) or document it here.